contracts/interfaces/IPeriphery.sol

```diff
@@ -8,11 +8,71 @@ import "../utils/CustomErrors.sol";
 import "../interfaces/IWETH.sol";
 
 interface IPeriphery is CustomErrors {
-    // events
+    // structs for events
+    struct PositionDetails {
+        IMarginEngine marginEngine;
+        address owner;
+        int24 tickLower;
+        int24 tickUpper;
+    }
+
+    struct SwapOutput {
+        int256 variableTokenDelta;
```

Comment:
do we also want to track fixedTokenDeltaBalanced as well as unbalanced just in case?

Reply:
done.